scripts/ci/prek/mypy_folder.py

```diff
@@ -116,7 +116,8 @@ def get_all_files(folder: str) -> list[str]:
         ) and not file.as_posix().endswith("src/airflow/providers/__init__.py"):
             files_to_check.append(file.relative_to(AIRFLOW_ROOT_PATH).as_posix())
     file_spec = "@/files/mypy_files.txt"
-    console.print(f"[info]Running mypy with {file_spec}")
+    if console:
```

Comment:
Inline comment to block accidental merge.

As TP mentioned - fallback printing is a good idea.